src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -41,6 +42,7 @@ internal sealed class SqlTriggerListener<T> : IListener, IScaleMonitor<SqlTrigge
         private readonly ScaleMonitorDescriptor _scaleMonitorDescriptor;
 
         private readonly IDictionary<TelemetryPropertyName, string> _telemetryProps = new Dictionary<TelemetryPropertyName, string>();
+        private readonly int _maxChangesPerWorker = DefaultMaxChangesPerWorker;
```

Comment:
Better not have two places for initialization. Keep `_maxChangesPerWorker` uninitialized here and inside the constructor, change to `this._maxChangesPerWorker = configuredMaxChangesPerWorker ?? DefaultMaxChangesPerWorker;`.